torch/utils/_pytree.py

```diff
@@ -337,7 +336,6 @@ def go(t):
     You can also directly use 'tree_map_only'
     """
     def deco(f: Callable[[T], Any]) -> Callable[[Any], Any]:
-        @functools.wraps(f)
```

Comment:
meh, ok. Ideally dynamo can deal with this though

Reply:
@kshitij12345 has an upcoming fix for this